lib/post_action_creator.rb

```diff
@@ -115,7 +115,9 @@ def perform
 
     # create meta topic / post if needed
     if @message.present? &&
-         %i[notify_moderators notify_user spam illegal].include?(@post_action_name)
+         (PostActionType.additional_message_types.keys | %i[spam illegal]).include?(
+           @post_action_name,
+         )
```

Comment:
Please note that before `create_message_creator` was called only for those types ` %i[notify_moderators notify_user spam illegal]`. It will be important for refactoring part of `create_message_creator